src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands.room;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class ListRoomCommand extends Command {
+    public static final String COMMAND_WORD = "listroom";
+    public static final String MESSAGE_SUCCESS = "Here are the list of rooms";
```

Comment:
Do you want to standardise the MESSAGE_SUCCESS with that of list patient command? Like "Listed all rooms."

Reply:
Okay.